assets/tests/components/shuttleMapPage.test.tsx

```diff
@@ -130,6 +130,47 @@ describe("Shuttle Map Page", () => {
     expect(result.asFragment()).toMatchSnapshot()
   })
 
+  test("changing selected shuttles re-enabled map centering", async () => {
+    const dispatch = jest.fn()
+    ;(useShuttleVehicles as jest.Mock).mockImplementationOnce(() => [shuttle])
+    const result = render(
+      <StateDispatchProvider
+        state={{ ...initialState, selectedShuttleRunIds: "all" }}
+        dispatch={dispatch}
+      >
+        <BrowserRouter>
+          <ShuttleMapPage />
+        </BrowserRouter>
+      </StateDispatchProvider>
+    )
+    await animationFramePromise()
+
+    // We can't directly use the recenter button to disable recentering, but zooming works.
+    // However, there is currently a bug causing us to not disable autocentering with a single
+    // click on one of the zoom buttons, instead requiring two.
+    await userEvent.click(screen.getByRole("button", { name: "Zoom in" }))
+    await animationFramePromise()
+    await userEvent.click(screen.getByRole("button", { name: "Zoom out" }))
```

Comment:
Could you use the selector helpers from `assets/tests/testHelpers/selectors/components/map.ts`?

Reply:
Updated in 84f7a5d8.